inlong-dashboard/src/pages/Clusters/NodeEditModal.tsx

```diff
@@ -106,6 +106,11 @@ const NodeEditModal: React.FC<NodeEditModalProps> = ({ id, type, clusterId, ...m
           max: 65535,
         },
       },
+      {
+        type: 'input',
```

Comment:
How many protocol types do we support?
Suggested changing the input to select box.